main.go

```diff
@@ -121,8 +123,14 @@ func main() {
 	trace(cmd)
 	cmd.Run()
 
+	strCmd := []string{"/usr/bin/docker", "build", "--pull=true","--rm=true", "-t", vargs.Repo, vargs.Context}
+	// Add file flag to cmd
+	if len(vargs.File) != 0 {
+		strCmd = append(append(strCmd[0:3],"-f="+vargs.File),strCmd[4:]...)
+	}
```

Comment:
I'm having a bit of difficulty reading the above code with the slicing/appending. Do you think we could re-write like this:

```Go
if len(vargs.File) == 0 {
	vargs.File = "Dockerfile"
}
if len(vargs.Context) == 0 {
	vargs.Context = "."
}
cmd = exec.Command("/usr/bin/docker", "build", "--pull=true", "--rm=true", "-t", "-f", vargs.File, vargs.Repo, vargs.Context)
```